pkg/collector/collector.go

```diff
@@ -415,10 +417,18 @@ func (cm *CollectorManager) CollectContainerEvents(id *ContainerId) {
 		}
 
 		// The name of the ApplicationProfile you're looking for.
-		appProfileName := fmt.Sprintf("pod-%s", id.PodName)
+		var appProfileName string
+		var namespace string
+		if cm.config.StoreNamespace != "" {
+			appProfileName = fmt.Sprintf("pod-%s.%s", id.PodName, id.Namespace)
+			namespace = cm.config.StoreNamespace
+		} else {
+			appProfileName = fmt.Sprintf("pod-%s", id.PodName)
+			namespace = id.Namespace
+		}
```

Comment:
It would be great to create a function that gets: 
* namespace
* kind
* name
and returns the application profile name (and another function that does the other way around) to save code

Reply:
done